### PR TITLE
feat: [SRM-13786]: implementing applied thresholds and UX review comments

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Implementing applied thresholds.
2. Addressing UX review items.
3. Updating the swagger after the backend fix and updating the pagination in the UI accordingly.

New Note

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.utils.ts [hunk 1]
```diff
@@ -93,13 +94,23 @@ export function transformMetricData(
       )
 
       // generating points for testHost
-      generatePointsForNodes(testData, testPoints, testAnalysisResult, analysisReason, nodeIdentifier)
+      generatePointsForNodes(
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.utils.ts:97]: pass arguments in an object

Comment (harshilbits) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.utils.ts]: done.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.utils.ts [hunk 2]
```diff
@@ -83,3 +83,16 @@ export function getControlDataType(
     return ''
   }
 }
+
+export const getVerificationType = (risk: RiskValues, getString: UseStringsReturn['getString']): string => {
+  switch (risk) {
+    case RiskValues.HEALTHY:
+      return getString('passed').toLocaleUpperCase()
+    case RiskValues.WARNING:
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.utils.ts:91]: RiskValues.FAILED and RiskValues.WARNING , can be clubbed

case RiskValues.WARNING:
case RiskValues.FAILED:
   return getString('failed').toLocaleUpperCase()

**Review by deepesh-ui — APPROVED** [review]

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.utils.ts (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.utils.ts:91]: done.

**Review by karansaraswat19 — DISMISSED** [review]

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss [hunk 3]
```diff
@@ -144,3 +144,7 @@
   line-height: 18px !important;
   color: var(--black) !important;
 }
+
+.metricDetails {
+  background: #f8f8fe !important;
```
Comment (praneshg239) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss:149]: use color from design system

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss [hunk 4]
```diff
@@ -7,9 +7,9 @@
 
 .main {
   width: 100% !important;
-
-  .transactionMetric {
-  }
+  border: 1px solid #effbff;
```
Comment (praneshg239) [inline src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss:10]: use color from design system

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss (hunk 3 above)
Comment (harshilbits) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss:149]: Couldn't find this color in the design system.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss (hunk 4 above)
Comment (harshilbits) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss:10]: same.

File: src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss (hunk 3 above)
Comment (praneshg239) [reply src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/components/DeploymentMetricsAnalysisRow/DeploymentMetricsAnalysisRow.module.scss:149]: If a color is used which is not in the design system, please check with Thani and tell him to give a nearest color from design system

**Review by praneshg239 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**